.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,16 +15,36 @@
 ### Before
 
 
-
 ### After
 
 
+## Release checklist
+
+[Release process documentation](../doc/how-to/perform-a-release.md)
+
+As part of our continuous deployment strategy we must ensure that this work is ready to be released at any point.
```

Comment:
You mean `ready to be released once merged`? It read to me like it needs to be ready to be released any time you push ("at any point")